### ISSUE TITLE
🐛 Wrong /lint/complexity/noCommaOperator for Svelte

### ISSUE THREAD

**Issue by wennerryle** [biomejs/biome#10265]
### Environment information

<details>

```bash

apps/web/src/routes/(authorized)/config/_components/WifiConnection.svelte:38:53 lint/complexity/noCommaOperator

  ⚠ The comma operator is disallowed.
  
    36 │                                <Toggle
    37 │                                        bind:pressed={
  > 38 │                                                () => viewmodel.network.status === 'connected',
       │                                                                                              ^
    39 │                                                (enable) => {
    40 │                                                        if (enable) {
  
  ℹ Its use is often confusing and obscures side effects.
  

apps/web/src/routes/(authorized)/devices/_components/DeviceCard.svelte:44:35 lint/complexity/noCommaOperator

  ⚠ The comma operator is disallowed.
  
    42 │        <div class="flex items-center justify-between px-4">
    43 │                <span>{title}</span>
  > 44 │                <Toggle bind:pressed={getToggled, setToggled} />
       │                                                ^
    45 │        </div>
    46 │        <div class="border-b border-gray-300"></div>
  
  ℹ Its use is often confusing and obscures side effects.
  

Checked 91 files in 61ms. No fixes applied.
Found 10 warnings.
Found 7 infos.
vpozdeev@bazzite:~/Projects/esp32wol-frontend$ 

```

</details>

### What happened?

[1. fst example](https://biomejs.dev/playground/?indentScriptAndStyle=true&lintRules=all&tab=formatter&pane=GritQL&code=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiAD4ACgAJAGkAbQBwAG8AcgB0ACAAewAgAFQAbwBnAGcAbABlACwAIABEAGkAYQBsAG8AZwBDAGwAbwBzAGUALAAgAEQAaQBhAGwAbwBnAE8AdgBlAHIAbABhAHkALAAgAEQAaQBhAGwAbwBnAEMAbwBuAHQAZQBuAHQALAAgAGMAbgAgAH0AIABmAHIAbwBtACAAJwAkAGwAaQBiAC8AYwBvAG0AcABvAG4AZQBuAHQAcwAnADsACgAJAGkAbQBwAG8AcgB0ACAAewAgAEQAaQBhAGwAbwBnACAAfQAgAGYAcgBvAG0AIAAnAGIAaQB0AHMALQB1AGkAJwA7AAoACQBpAG0AcABvAHIAdAAgAHsAIABiAHUAdAB0AG8AbgAsACAAdABlAHgAdABhAHIAZQBhACAAfQAgAGYAcgBvAG0AIAAnACQAbABpAGIALwBjAG8AbQBwAG8AbgBlAG4AdABzAC8AcwB0AHkAbABlAHMAJwA7AAoACgAJAGkAbQBwAG8AcgB0ACAAUwBxAHUAYQByAGUAUABlAG4AIABmAHIAbwBtACAAJwBAAGwAdQBjAGkAZABlAC8AcwB2AGUAbAB0AGUALwBpAGMAbwBuAHMALwBzAHEAdQBhAHIAZQAtAHAAZQBuACcAOwAKAAkAaQBtAHAAbwByAHQAIABUAHIAYQBzAGgAMgAgAGYAcgBvAG0AIAAnAEAAbAB1AGMAaQBkAGUALwBzAHYAZQBsAHQAZQAvAGkAYwBvAG4AcwAvAHQAcgBhAHMAaAAtADIAJwA7AAoACQBpAG0AcABvAHIAdAAgAHsAIABzAHQAbwBwAEUAdgBlAG4AdAAgAH0AIABmAHIAbwBtACAAJwAkAGwAaQBiAC8AdQB0AGkAbABzACcAOwAKAAoACQBpAG4AdABlAHIAZgBhAGMAZQAgAEUAZABpAHQAYQBiAGwAZQBEAGUAdgBpAGMAZQBJAG4AZgBvACAAewAKAAkACQBhAGwAaQBhAHMAOgAgAHMAdAByAGkAbgBnADsACgAJAAkAaQBwADoAIABzAHQAcgBpAG4AZwA7AAoACQAJAGIAcgBvAGEAZABjAGEAcwB0ADoAIABzAHQAcgBpAG4AZwA7AAoACQB9AAoACgAJAGkAbgB0AGUAcgBmAGEAYwBlACAAUAByAG8AcABzACAAewAKAAkACQB0AGkAdABsAGUAOgAgAHMAdAByAGkAbgBnADsACgAJAAkAaQBwADoAIABzAHQAcgBpAG4AZwA7AAoACQAJAGIAcgBvAGEAZABjAGEAcwB0ADoAIABzAHQAcgBpAG4AZwA7AAoACQAJAG0AYQBjADoAIABzAHQAcgBpAG4AZwA7AAoACQAJAGcAZQB0AFQAbwBnAGcAbABlAGQAOgAgACgAKQAgAD0APgAgAGIAbwBvAGwAZQBhAG4AOwAKAAkACQBzAGUAdABUAG8AZwBnAGwAZQBkADoAIAAoAGUAbgBhAGIAbABlAGQAOgAgAGIAbwBvAGwAZQBhAG4AKQAgAD0APgAgAHYAbwBpAGQAOwAKAAkACQBvAG4ARABlAHYAaQBjAGUARQBkAGkAdAA6ACAAKABkAGUAdgBpAGMAZQBJAG4AZgBvADoAIABFAGQAaQB0AGEAYgBsAGUARABlAHYAaQBjAGUASQBuAGYAbwApACAAPQA%2BACAAdgBvAGkAZAA7AAoACQAJAG8AbgBEAGUAbABlAHQAZQA6ACAAKAApACAAPQA%2BACAAdgBvAGkAZAA7AAoACQB9AAoACgAJAGMAbwBuAHMAdAAgAHsAIAB0AGkAdABsAGUALAAgAGkAcAAsACAAYgByAG8AYQBkAGMAYQBzAHQALAAgAG0AYQBjACwAIABnAGUAdABUAG8AZwBnAGwAZQBkACwAIABzAGUAdABUAG8AZwBnAGwAZQBkACwAIABvAG4ARABlAHYAaQBjAGUARQBkAGkAdAAsACAAbwBuAEQAZQBsAGUAdABlACAAfQA6ACAAUAByAG8AcABzACAAPQAKAAkACQAkAHAAcgBvAHAAcwAoACkAOwAKAAoACQBsAGUAdAAgAGQAaQBhAGwAbwBnAE8AcABlAG4AIAA9ACAAJABzAHQAYQB0AGUAKABmAGEAbABzAGUAKQA7AAoACgAJAGMAbwBuAHMAdAAgAGQAaQBhAGwAbwBnAEYAaQBlAGwAZABzADoAIABFAGQAaQB0AGEAYgBsAGUARABlAHYAaQBjAGUASQBuAGYAbwAgAD0AIAAkAHMAdABhAHQAZQAoAHsACgAJAAkAYQBsAGkAYQBzADoAIAAnACcALAAKAAkACQBpAHAAOgAgACcAJwAsAAoACQAJAGIAcgBvAGEAZABjAGEAcwB0ADoAIAAnACcACgAJAH0AKQA7AAoAPAAvAHMAYwByAGkAcAB0AD4ACgAKADwAZABpAHYACgAJAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGYAbABlAHgALQBjAG8AbAAgAGcAYQBwAC0ANAAgAG8AdgBlAHIAZgBsAG8AdwAtAGMAbABpAHAAIAByAG8AdQBuAGQAZQBkAC0AeABsACAAYgBnAC0AZwByAGEAeQAtADUAMAAgAHAAdAAtADQAIAB0AGUAeAB0AC0AYgBhAHMAZQAgAGwAZQBhAGQAaQBuAGcALQA2ACAAZgBvAG4AdAAtAG0AZQBkAGkAdQBtACAAbABnADoAdABlAHgAdAAtAGwAZwAgAGwAZwA6AGwAZQBhAGQAaQBuAGcALQA3ACIACgA%2BAAoACQA8AGQAaQB2ACAAYwBsAGEAcwBzAD0AIgBmAGwAZQB4ACAAaQB0AGUAbQBzAC0AYwBlAG4AdABlAHIAIABqAHUAcwB0AGkAZgB5AC0AYgBlAHQAdwBlAGUAbgAgAHAAeAAtADQAIgA%2BAAoACQAJADwAcwBwAGEAbgA%2BAHsAdABpAHQAbABlAH0APAAvAHMAcABhAG4APgAKAAkACQA8AFQAbwBnAGcAbABlACAAYgBpAG4AZAA6AHAAcgBlAHMAcwBlAGQAPQB7AGcAZQB0AFQAbwBnAGcAbABlAGQALAAgAHMAZQB0AFQAbwBnAGcAbABlAGQAfQAgAC8APgAKAAkAPAAvAGQAaQB2AD4ACgAJADwAZABpAHYAIABjAGwAYQBzAHMAPQAiAGIAbwByAGQAZQByAC0AYgAgAGIAbwByAGQAZQByAC0AZwByAGEAeQAtADMAMAAwACIAPgA8AC8AZABpAHYAPgAKAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGYAbABlAHgALQBjAG8AbAAgAGcAYQBwAC0AMgAgAHAAeAAtADQAIgA%2BAAoACQAJADwAcwBwAGEAbgA%2BAEkAUAA6ACAAewBpAHAAfQA8AC8AcwBwAGEAbgA%2BAAoACQAJADwAcwBwAGEAbgAgAGMAbABhAHMAcwA9ACIAdABlAHgAdAAtAGcAcgBhAHkALQA3ADAAMAAiAD4AQgByAG8AYQBkAGMAYQBzAHQAOgAgAHsAYgByAG8AYQBkAGMAYQBzAHQAfQA8AC8AcwBwAGEAbgA%2BAAoACQAJADwAcwBwAGEAbgAgAGMAbABhAHMAcwA9ACIAdABlAHgAdAAtAGcAcgBhAHkALQA3ADAAMAAiAD4ATQBBAEMAOgAgAHsAbQBhAGMAfQA8AC8AcwBwAGEAbgA%2BAAoACQA8AC8AZABpAHYAPgAKAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGgALQAxADIAIABsAGcAOgBoAC0AMQA1ACIAPgAKAAkACQA8AEQAaQBhAGwAbwBnAC4AUgBvAG8AdAAgAGIAaQBuAGQAOgBvAHAAZQBuAD0AewBkAGkAYQBsAG8AZwBPAHAAZQBuAH0APgAKAAkACQAJADwARABpAGEAbABvAGcALgBUAHIAaQBnAGcAZQByAAoACQAJAAkACQBjAGwAYQBzAHMAPQAiAGYAbABlAHgAIABmAGwAZQB4AC0AMQAgAGMAdQByAHMAbwByAC0AcABvAGkAbgB0AGUAcgAgAGkAdABlAG0AcwAtAGMAZQBuAHQAZQByACAAagB1AHMAdABpAGYAeQAtAGMAZQBuAHQAZQByACAAZwBhAHAALQAyAC4ANQAgAGIAbwByAGQAZQByAC0AdAAgAGIAbwByAGQAZQByAC0AcgAgAGIAbwByAGQAZQByAC0AZwByAGEAeQAtADMAMAAwACAAbwB1AHQAbABpAG4AZQAtADAAIAB0AHIAYQBuAHMAaQB0AGkAbwBuAC0AYwBvAGwAbwByAHMAIABoAG8AdgBlAHIAOgBiAGcALQBnAHIAYQB5AC0AMgAwADAAIABmAG8AYwB1AHMAOgBiAGcALQBnAHIAYQB5AC0AMgAwADAAIgAKAAkACQAJAD4ACgAJAAkACQAJADwAUwBxAHUAYQByAGUAUABlAG4AIAAvAD4ACgAJAAkACQAJACAENQQ0BDAEOgRCBDgEQAQ%2BBDIEMARCBEwECgAJAAkACQA8AC8ARABpAGEAbABvAGcALgBUAHIAaQBnAGcAZQByAD4ACgAJAAkACQA8AEQAaQBhAGwAbwBnAC4AUABvAHIAdABhAGwAPgAKAAkACQAJAAkAPABEAGkAYQBsAG8AZwBPAHYAZQByAGwAYQB5ACAALwA%2BAAoACQAJAAkACQA8AEQAaQBhAGwAbwBnAEMAbwBuAHQAZQBuAHQAPgAKAAkACQAJAAkACQA8AEQAaQBhAGwAbwBnAEMAbABvAHMAZQAgAC8APgAKAAkACQAJAAkACQA8AGYAbwByAG0AIABjAGwAYQBzAHMAPQAiAGYAbABlAHgAIABmAGwAZQB4AC0AYwBvAGwAIABnAGEAcAAtADIALgA1ACIAIABvAG4AcwB1AGIAbQBpAHQAPQB7AHMAdABvAHAARQB2AGUAbgB0AH0APgAKAAkACQAJAAkACQAJADwAaAAyACAAYwBsAGEAcwBzAD0AIgB0AGUAeAB0AC0AMgB4AGwAIABsAGUAYQBkAGkAbgBnAC0AOAAgAGYAbwBuAHQALQBtAGUAZABpAHUAbQAiAD4AIAQ1BDQEMAQ6BEIEOARABD4EMgQwBD0EOAQ1BDwAYgByACAALwA%2BAEMEQQRCBEAEPgQ5BEEEQgQyBDAEPAAvAGgAMgA%2BAAoACQAJAAkACQAJAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9AHsAdABlAHgAdABhAHIAZQBhAC4AYwBvAG4AdABhAGkAbgBlAHIAfQA%2BAAoACQAJAAkACQAJAAkACQA8AGwAYQBiAGUAbAAgAGYAbwByAD0AIgBhAGwAaQBhAHMAIgA%2BAEEAbABpAGEAcwA8AC8AbABhAGIAZQBsAD4ACgAJAAkACQAJAAkACQAJADwAaQBuAHAAdQB0AAoACQAJAAkACQAJAAkACQAJAHQAeQBwAGUAPQAiAHQAZQB4AHQAIgAKAAkACQAJAAkACQAJAAkACQBpAGQAPQAiAGEAbABpAGEAcwAiAAoACQAJAAkACQAJAAkACQAJAGMAbABhAHMAcwA9AHsAdABlAHgAdABhAHIAZQBhAC4AaQBuAHAAdQB0AH0ACgAJAAkACQAJAAkACQAJAAkAYgBpAG4AZAA6AHYAYQBsAHUAZQA9AHsAZABpAGEAbABvAGcARgBpAGUAbABkAHMALgBhAGwAaQBhAHMAfQAKAAkACQAJAAkACQAJAAkACQBwAGwAYQBjAGUAaABvAGwAZABlAHIAPQAiAEEAbABpAGEAcwAiAAoACQAJAAkACQAJAAkACQAvAD4ACgAJAAkACQAJAAkACQA8AC8AZABpAHYAPgAKAAkACQAJAAkACQAJADwAZABpAHYAIABjAGwAYQBzAHMAPQB7AHQAZQB4AHQAYQByAGUAYQAuAGMAbwBuAHQAYQBpAG4AZQByAH0APgAKAAkACQAJAAkACQAJAAkAPABsAGEAYgBlAGwAIABmAG8AcgA9ACIAaQBwACIAPgBJAFAAPAAvAGwAYQBiAGUAbAA%2BAAoACQAJAAkACQAJAAkACQA8AGkAbgBwAHUAdAAKAAkACQAJAAkACQAJAAkACQB0AHkAcABlAD0AIgB0AGUAeAB0ACIACgAJAAkACQAJAAkACQAJAAkAaQBkAD0AIgBpAHAAIgAKAAkACQAJAAkACQAJAAkACQBjAGwAYQBzAHMAPQB7AHQAZQB4AHQAYQByAGUAYQAuAGkAbgBwAHUAdAB9AAoACQAJAAkACQAJAAkACQAJAGIAaQBuAGQAOgB2AGEAbAB1AGUAPQB7AGQAaQBhAGwAbwBnAEYAaQBlAGwAZABzAC4AaQBwAH0ACgAJAAkACQAJAAkACQAJAAkAcABsAGEAYwBlAGgAbwBsAGQAZQByAD0AIgBJAFAAIgAKAAkACQAJAAkACQAJAAkALwA%2BAAoACQAJAAkACQAJAAkAPAAvAGQAaQB2AD4ACgAJAAkACQAJAAkACQA8AGQAaQB2ACAAYwBsAGEAcwBzAD0AewB0AGUAeAB0AGEAcgBlAGEALgBjAG8AbgB0AGEAaQBuAGUAcgB9AD4ACgAJAAkACQAJAAkACQAJADwAbABhAGIAZQBsACAAZgBvAHIAPQAiAGIAcgBvAGEAZABjAGEAcwB0ACIAPgBCAHIAbwBhAGQAYwBhAHMAdAA8AC8AbABhAGIAZQBsAD4ACgAJAAkACQAJAAkACQAJADwAaQBuAHAAdQB0AAoACQAJAAkACQAJAAkACQAJAHQAeQBwAGUAPQAiAHQAZQB4AHQAIgAKAAkACQAJAAkACQAJAAkACQBpAGQAPQAiAGIAcgBvAGEAZABjAGEAcwB0ACIACgAJAAkACQAJAAkACQAJAAkAYwBsAGEAcwBzAD0AewB0AGUAeAB0AGEAcgBlAGEALgBpAG4AcAB1AHQAfQAKAAkACQAJAAkACQAJAAkACQBiAGkAbgBkADoAdgBhAGwAdQBlAD0AewBkAGkAYQBsAG8AZwBGAGkAZQBsAGQAcwAuAGIAcgBvAGEAZABjAGEAcwB0AH0ACgAJAAkACQAJAAkACQAJAAkAcABsAGEAYwBlAGgAbwBsAGQAZQByAD0AIgBCAHIAbwBhAGQAYwBhAHMAdAAiAAoACQAJAAkACQAJAAkACQAvAD4ACgAJAAkACQAJAAkACQA8AC8AZABpAHYAPgAKAAkACQAJAAkACQAJADwAYgB1AHQAdABvAG4ACgAJAAkACQAJAAkACQAJAHQAeQBwAGUAPQAiAGIAdQB0AHQAbwBuACIACgAJAAkACQAJAAkACQAJAGMAbABhAHMAcwA9AHsAYwBuACgAYgB1AHQAdABvAG4ALgBiAGEAcwBlACwAIABiAHUAdAB0AG8AbgAuAHAAcgBpAG0AYQByAHkALAAgAGIAdQB0AHQAbwBuAC4AbABnACkAfQAKAAkACQAJAAkACQAJAAkAbwBuAGMAbABpAGMAawA9AHsAKAApACAAPQA%2BACAAewAKAAkACQAJAAkACQAJAAkACQBvAG4ARABlAHYAaQBjAGUARQBkAGkAdAAoAGQAaQBhAGwAbwBnAEYAaQBlAGwAZABzACkAOwAKAAkACQAJAAkACQAJAAkAfQB9AAoACQAJAAkACQAJAAkAPgAKAAkACQAJAAkACQAJAAkAIQQ%2BBEUEQAQwBD0EOARCBEwECgAJAAkACQAJAAkACQA8AC8AYgB1AHQAdABvAG4APgAKAAkACQAJAAkACQA8AC8AZgBvAHIAbQA%2BAAoACQAJAAkACQA8AC8ARABpAGEAbABvAGcAQwBvAG4AdABlAG4AdAA%2BAAoACQAJAAkAPAAvAEQAaQBhAGwAbwBnAC4AUABvAHIAdABhAGwAPgAKAAkACQA8AC8ARABpAGEAbABvAGcALgBSAG8AbwB0AD4ACgAJAAkAPABiAHUAdAB0AG8AbgAKAAkACQAJAHQAeQBwAGUAPQAiAGIAdQB0AHQAbwBuACIACgAJAAkACQBjAGwAYQBzAHMAPQAiAGYAbABlAHgAIAB3AC0AMQA1ACAAYwB1AHIAcwBvAHIALQBwAG8AaQBuAHQAZQByACAAaQB0AGUAbQBzAC0AYwBlAG4AdABlAHIAIABqAHUAcwB0AGkAZgB5AC0AYwBlAG4AdABlAHIAIABiAG8AcgBkAGUAcgAtAHQAIABiAG8AcgBkAGUAcgAtAGcAcgBhAHkALQAzADAAMAAgAG8AdQB0AGwAaQBuAGUALQAwACAAdAByAGEAbgBzAGkAdABpAG8AbgAtAGMAbwBsAG8AcgBzACAAaABvAHYAZQByADoAYgBnAC0AZwByAGEAeQAtADIAMAAwACAAZgBvAGMAdQBzADoAYgBnAC0AZwByAGEAeQAtADIAMAAwACIACgAJAAkACQBvAG4AYwBsAGkAYwBrAD0AewBvAG4ARABlAGwAZQB0AGUAfQAKAAkACQA%2BAAoACQAJAAkAPABzAHAAYQBuACAAYwBsAGEAcwBzAD0AIgBzAHIALQBvAG4AbAB5ACIAPgAjBDQEMAQ7BDgEQgRMBDwALwBzAHAAYQBuAD4ACgAJAAkACQA8AFQAcgBhAHMAaAAyACAALwA%2BAAoACQAJADwALwBiAHUAdAB0AG8AbgA%2BAAoACQA8AC8AZABpAHYAPgAKADwALwBkAGkAdgA%2BAAoA&language=svelte)

[2. snd example](https://biomejs.dev/playground/?indentScriptAndStyle=true&lintRules=all&tab=formatter&pane=GritQL&code=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiAD4ACgAJAGkAbQBwAG8AcgB0ACAAdAB5AHAAZQAgAHsAIABXAGkAZgBpAEMAbwBuAG4AZQBjAHQAaQBvAG4AVgBpAGUAdwBNAG8AZABlAGwAIAB9ACAAZgByAG8AbQAgACcALgAvAFcAaQBmAGkAQwBvAG4AbgBlAGMAdABpAG8AbgBWAGkAZQB3AE0AbwBkAGUAbAAuAHMAdgBlAGwAdABlACcAOwAKAAkAaQBtAHAAbwByAHQAIAB7ACAAYwBuACwAIABQAGEAcwBzAHcAbwByAGQARQB5AGUAcwBCAHUAdAB0AG8AbgAsACAAUABhAHMAcwB3AG8AcgBkAFQAbwBnAGcAbABlACwAIABUAG8AZwBnAGwAZQAgAH0AIABmAHIAbwBtACAAJwAkAGwAaQBiAC8AYwBvAG0AcABvAG4AZQBuAHQAcwAnADsACgAJAGkAbQBwAG8AcgB0ACAAewAgAEEAYwBjAG8AcgBkAGkAbwBuACAAfQAgAGYAcgBvAG0AIAAnAGIAaQB0AHMALQB1AGkAJwA7AAoACQBpAG0AcABvAHIAdAAgAFMAUwBJAEQASQBuAGQAaQBjAGEAdABvAHIAIABmAHIAbwBtACAAJwAkAGwAaQBiAC8AYwBvAG0AcABvAG4AZQBuAHQAcwAvAGIAdQBzAGkAbgBlAHMAcwAvAFMAUwBJAEQASQBuAGQAaQBjAGEAdABvAHIALgBzAHYAZQBsAHQAZQAnADsACgAJAGkAbQBwAG8AcgB0ACAAewAgAGIAdQB0AHQAbwBuACwAIAB0AGUAeAB0AGEAcgBlAGEAIAB9ACAAZgByAG8AbQAgACcAJABsAGkAYgAvAGMAbwBtAHAAbwBuAGUAbgB0AHMALwBzAHQAeQBsAGUAcwAnADsACgAJAGkAbQBwAG8AcgB0ACAAewAgAHMAdABvAHAARQB2AGUAbgB0ACAAfQAgAGYAcgBvAG0AIAAnACQAbABpAGIALwB1AHQAaQBsAHMAJwA7AAoACQBpAG0AcABvAHIAdAAgAEwAbwBhAGQAZQByAEMAaQByAGMAbABlACAAZgByAG8AbQAgACcAQABsAHUAYwBpAGQAZQAvAHMAdgBlAGwAdABlAC8AaQBjAG8AbgBzAC8AbABvAGEAZABlAHIALQBjAGkAcgBjAGwAZQAnADsACgAKAAkAaQBuAHQAZQByAGYAYQBjAGUAIABQAHIAbwBwAHMAIAB7AAoACQAJAHYAaQBlAHcAbQBvAGQAZQBsADoAIABXAGkAZgBpAEMAbwBuAG4AZQBjAHQAaQBvAG4AVgBpAGUAdwBNAG8AZABlAGwAOwAKAAkAfQAKAAoACQBjAG8AbgBzAHQAIAB7ACAAdgBpAGUAdwBtAG8AZABlAGwAIAB9ADoAIABQAHIAbwBwAHMAIAA9ACAAJABwAHIAbwBwAHMAKAApADsACgAKAAkAYwBvAG4AcwB0ACAAcABhAHMAcwB3AG8AcgBkACAAPQAgAG4AZQB3ACAAUABhAHMAcwB3AG8AcgBkAFQAbwBnAGcAbABlACgAKQA7AAoAPAAvAHMAYwByAGkAcAB0AD4ACgAKADwAQQBjAGMAbwByAGQAaQBvAG4ALgBJAHQAZQBtAAoACQB2AGEAbAB1AGUAPQB7AHYAaQBlAHcAbQBvAGQAZQBsAC4AbgBlAHQAdwBvAHIAawAuAG0AYQBjAH0ACgAJAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAHcALQBmAHUAbABsACAAZgBsAGUAeAAtAGMAbwBsACAAagB1AHMAdABpAGYAeQAtAGIAZQB0AHcAZQBlAG4AIAB0AGUAeAB0AC0AYgBhAHMAZQAgAGwAZQBhAGQAaQBuAGcALQA2ACAAZgBvAG4AdAAtAG0AZQBkAGkAdQBtACAAZABhAHQAYQAtAFsAcwB0AGEAdABlAD0AbwBwAGUAbgBdADoAYgBnAC0AZwByAGEAeQAtADEAMAAwACIACgA%2BAAoACQA8AEEAYwBjAG8AcgBkAGkAbwBuAC4ASABlAGEAZABlAHIAIABjAGwAYQBzAHMAPQAiAGcAcgBvAHUAcAAvAGEAYwBjAG8AcgBkAGkAbwBuAC0AaABlAGEAZABlAHIAIgA%2BAAoACQAJADwAZABpAHYACgAJAAkACQBjAGwAYQBzAHMAPQAiAGYAbABlAHgAIABwAHgALQA0ACAAcAB5AC0AMQAgAGcAcgBvAHUAcAAtAGQAYQB0AGEALQBbAHMAdABhAHQAZQA9AGMAbABvAHMAZQBdADoAZABlAGwAYQB5AC0ANwA1ACAAZwByAG8AdQBwAC0AZABhAHQAYQAtAFsAcwB0AGEAdABlAD0AbwBwAGUAbgBdADoAcAB5AC0AMgAgAGgAbwB2AGUAcgA6AGIAZwAtAGcAcgBhAHkALQAxADAAMAAiAAoACQAJAD4ACgAJAAkACQA8AEEAYwBjAG8AcgBkAGkAbwBuAC4AVAByAGkAZwBnAGUAcgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAHcALQBmAHUAbABsACAAYwB1AHIAcwBvAHIALQBwAG8AaQBuAHQAZQByACAAaQB0AGUAbQBzAC0AYwBlAG4AdABlAHIAIABqAHUAcwB0AGkAZgB5AC0AYgBlAHQAdwBlAGUAbgAiAD4ACgAJAAkACQAJADwAcwBwAGEAbgA%2BAAoACQAJAAkACQAJAHsAdgBpAGUAdwBtAG8AZABlAGwALgBuAGUAdAB3AG8AcgBrAC4AcwBzAGkAZAB9AAoACQAJAAkACQA8AC8AcwBwAGEAbgA%2BAAoACQAJAAkAPAAvAEEAYwBjAG8AcgBkAGkAbwBuAC4AVAByAGkAZwBnAGUAcgA%2BAAoACQAJAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGkAdABlAG0AcwAtAGMAZQBuAHQAZQByACAAZwBhAHAALQAyACIAPgAKAAkACQAJAAkAewAjAGkAZgAgAHYAaQBlAHcAbQBvAGQAZQBsAC4AbABvAGEAZABpAG4AZwB9AAoACQAJAAkACQAJADwATABvAGEAZABlAHIAQwBpAHIAYwBsAGUAIABjAGwAYQBzAHMAPQAiAGEAbgBpAG0AYQB0AGUALQBzAHAAaQBuACIAIABzAHQAcgBvAGsAZQBXAGkAZAB0AGgAPQAiADEAIgAgAC8APgAKAAkACQAJAAkAewAvAGkAZgB9AAoACQAJAAkACQA8AFQAbwBnAGcAbABlAAoACQAJAAkACQAJAGIAaQBuAGQAOgBwAHIAZQBzAHMAZQBkAD0AewAKAAkACQAJAAkACQAJACgAKQAgAD0APgAgAHYAaQBlAHcAbQBvAGQAZQBsAC4AbgBlAHQAdwBvAHIAawAuAHMAdABhAHQAdQBzACAAPQA9AD0AIAAnAGMAbwBuAG4AZQBjAHQAZQBkACcALAAKAAkACQAJAAkACQAJACgAZQBuAGEAYgBsAGUAKQAgAD0APgAgAHsACgAJAAkACQAJAAkACQAJAGkAZgAgACgAZQBuAGEAYgBsAGUAKQAgAHsACgAJAAkACQAJAAkACQAJAAkAdgBpAGUAdwBtAG8AZABlAGwALgBjAG8AbgBuAGUAYwB0ACgAKQA7AAoACQAJAAkACQAJAAkACQB9ACAAZQBsAHMAZQAgAHsACgAJAAkACQAJAAkACQAJAAkAdgBpAGUAdwBtAG8AZABlAGwALgBkAGkAcwBjAG8AbgBuAGUAYwB0ACgAKQA7AAoACQAJAAkACQAJAAkACQB9AAoACQAJAAkACQAJAAkAfQAKAAkACQAJAAkACQB9AAoACQAJAAkACQAvAD4ACgAJAAkACQA8AC8AZABpAHYAPgAKAAkACQA8AC8AZABpAHYAPgAKAAkAPAAvAEEAYwBjAG8AcgBkAGkAbwBuAC4ASABlAGEAZABlAHIAPgAKAAkAPABBAGMAYwBvAHIAZABpAG8AbgAuAEMAbwBuAHQAZQBuAHQACgAJAAkAYwBsAGEAcwBzAD0AIgBvAHYAZQByAGYAbABvAHcALQBoAGkAZABkAGUAbgAgAGIAZwAtAGcAcgBhAHkALQAxADAAMAAgAGQAYQB0AGEALQBbAHMAdABhAHQAZQA9AGMAbABvAHMAZQBkAF0AOgBhAG4AaQBtAGEAdABlAC0AYQBjAGMAbwByAGQAaQBvAG4ALQB1AHAAIABkAGEAdABhAC0AWwBzAHQAYQB0AGUAPQBvAHAAZQBuAF0AOgBhAG4AaQBtAGEAdABlAC0AYQBjAGMAbwByAGQAaQBvAG4ALQBkAG8AdwBuACIACgAJAD4ACgAJAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGYAbABlAHgALQBjAG8AbAAgAGcAYQBwAC0AMgAgAHAAeAAtADQAIABwAHQALQAyACAAcABiAC0ANAAiAD4ACgAJAAkACQA8AHMAcABhAG4AIABjAGwAYQBzAHMAPQAiAHQAZQB4AHQALQBnAHIAYQB5AC0ANwAwADAAIgA%2BAAoACQAJAAkACQB7AHYAaQBlAHcAbQBvAGQAZQBsAC4AbgBlAHQAdwBvAHIAawAuAHAAcgBvAHQAbwBjAG8AbAB9ACAAfAAgAHsAdgBpAGUAdwBtAG8AZABlAGwALgBuAGUAdAB3AG8AcgBrAC4AcwBlAGMAdQByAGkAdAB5AH0ACgAJAAkACQA8AC8AcwBwAGEAbgA%2BAAoACQAJAAkAPABkAGkAdgAgAGMAbABhAHMAcwA9ACIAZgBsAGUAeAAgAGkAdABlAG0AcwAtAGMAZQBuAHQAZQByACAAZwBhAHAALQAyAC4ANQAiAD4ACgAJAAkACQAJADwAUwBTAEkARABJAG4AZABpAGMAYQB0AG8AcgAgAGwAZQB2AGUAbAA9AHsAdgBpAGUAdwBtAG8AZABlAGwALgBzAGkAZwBuAGEAbABMAGUAdgBlAGwAfQAgAC8APgAKAAkACQAJAAkAPABzAHAAYQBuACAAYwBsAGEAcwBzAD0AIgB0AGUAeAB0AC0AZwByAGEAeQAtADcAMAAwACIAPgAKAAkACQAJAAkACQB7AHYAaQBlAHcAbQBvAGQAZQBsAC4AbgBlAHQAdwBvAHIAawAuAHMAaQBnAG4AYQBsAFMAdAByAGUAbgBnAHQAaAB9ACAAZABCAG0ACgAJAAkACQAJADwALwBzAHAAYQBuAD4ACgAJAAkACQA8AC8AZABpAHYAPgAKAAkACQAJAHsAIwBpAGYAIAB2AGkAZQB3AG0AbwBkAGUAbAAuAG4AZQB0AHcAbwByAGsALgBpAHMAUAByAG8AdABlAGMAdABlAGQAfQAKAAkACQAJAAkAPABmAG8AcgBtACAAYwBsAGEAcwBzAD0AIgBjAG8AbgB0AGUAbgB0AHMAIgAgAG8AbgBzAHUAYgBtAGkAdAA9AHsAcwB0AG8AcABFAHYAZQBuAHQAfQA%2BAAoACQAJAAkACQAJADwAZABpAHYAIABjAGwAYQBzAHMAPQB7AHQAZQB4AHQAYQByAGUAYQAuAGMAbwBuAHQAYQBpAG4AZQByAH0APgAKAAkACQAJAAkACQAJADwAbABhAGIAZQBsACAAZgBvAHIAPQB7AHYAaQBlAHcAbQBvAGQAZQBsAC4AbgBlAHQAdwBvAHIAawAuAG0AYQBjACAAKwAgACcAXwBwAGEAcwBzAHcAbwByAGQAJwB9AD4AIAAfBDAEQAQ%2BBDsETAQgADwALwBsAGEAYgBlAGwAPgAKAAkACQAJAAkACQAJADwAZABpAHYAIABjAGwAYQBzAHMAPQB7AHQAZQB4AHQAYQByAGUAYQAuAGkAYwBvAG4ASQBuAHAAdQB0AEMAbwBuAHQAYQBpAG4AZQByAH0APgAKAAkACQAJAAkACQAJAAkAPABpAG4AcAB1AHQACgAJAAkACQAJAAkACQAJAAkAaQBkAD0AewB2AGkAZQB3AG0AbwBkAGUAbAAuAG4AZQB0AHcAbwByAGsALgBtAGEAYwAgACsAIAAnAF8AcABhAHMAcwB3AG8AcgBkACcAfQAKAAkACQAJAAkACQAJAAkACQB0AHkAcABlAD0AewBwAGEAcwBzAHcAbwByAGQALgB0AHkAcABlAH0ACgAJAAkACQAJAAkACQAJAAkAYwBsAGEAcwBzAD0AewBjAG4AKAB0AGUAeAB0AGEAcgBlAGEALgBpAGMAbwBuAEkAbgBwAHUAdAAsACAAJwB3AC0AZgB1AGwAbAAgAGIAZwAtAHcAaABpAHQAZQAnACkAfQAKAAkACQAJAAkACQAJAAkACQBhAHUAdABvAGMAbwBtAHAAbABlAHQAZQA9ACIAbwBmAGYAIgAKAAkACQAJAAkACQAJAAkACQBiAGkAbgBkADoAdgBhAGwAdQBlAD0AewB2AGkAZQB3AG0AbwBkAGUAbAAuAHAAYQBzAHMAdwBvAHIAZAB9AAoACQAJAAkACQAJAAkACQAvAD4ACgAJAAkACQAJAAkACQAJADwAUABhAHMAcwB3AG8AcgBkAEUAeQBlAHMAQgB1AHQAdABvAG4AIABvAG4AYwBsAGkAYwBrAD0AewBwAGEAcwBzAHcAbwByAGQALgBjAGwAaQBjAGsASABhAG4AZABsAGUAcgB9ACAALwA%2BAAoACQAJAAkACQAJAAkAPAAvAGQAaQB2AD4ACgAJAAkACQAJAAkACQA8AHAAIABjAGwAYQBzAHMAPQB7AHQAZQB4AHQAYQByAGUAYQAuAGgAZQBsAHAAVABlAHgAdAB9AD4AIQQ1BEIETAQgADcEMARJBDgESQQ1BD0EMAQuACAAEgQyBDUENAQ4BEIENQQgAD8EMARABD4EOwRMBCAARwRCBD4EMQRLBCAAPwRABD4ENAQ%2BBDsENgQ4BEIETAQuADwALwBwAD4ACgAJAAkACQAJAAkACQB7ACMAaQBmACAAdgBpAGUAdwBtAG8AZABlAGwALgBlAHIAcgBvAHIAfQAKAAkACQAJAAkACQAJAAkAPABwACAAYwBsAGEAcwBzAD0AewB0AGUAeAB0AGEAcgBlAGEALgBlAHIAcgBvAHIAVABlAHgAdAB9AD4ACgAJAAkACQAJAAkACQAJAAkAewB2AGkAZQB3AG0AbwBkAGUAbAAuAGUAcgByAG8AcgB9AAoACQAJAAkACQAJAAkACQA8AC8AcAA%2BAAoACQAJAAkACQAJAAkAewAvAGkAZgB9AAoACQAJAAkACQAJADwALwBkAGkAdgA%2BAAoACQAJAAkACQAJADwAYgB1AHQAdABvAG4ACgAJAAkACQAJAAkACQB0AHkAcABlAD0AIgBiAHUAdAB0AG8AbgAiAAoACQAJAAkACQAJAAkAZABpAHMAYQBiAGwAZQBkAD0AewB2AGkAZQB3AG0AbwBkAGUAbAAuAGwAbwBhAGQAaQBuAGcAfQAKAAkACQAJAAkACQAJAGMAbABhAHMAcwA9AHsAWwBiAHUAdAB0AG8AbgAuAGIAYQBzAGUALAAgAGIAdQB0AHQAbwBuAC4AbABnACwAIABiAHUAdAB0AG8AbgAuAHAAcgBpAG0AYQByAHkAXQB9AAoACQAJAAkACQAJAAkAbwBuAGMAbABpAGMAawA9AHsAdgBpAGUAdwBtAG8AZABlAGwALgBjAG8AbgBuAGUAYwB0AH0ACgAJAAkACQAJAAkAPgAKAAkACQAJAAkACQAJAHsAIwBpAGYAIAB2AGkAZQB3AG0AbwBkAGUAbAAuAG4AZQB0AHcAbwByAGsALgBzAHQAYQB0AHUAcwAgAD0APQA9ACAAJwBjAG8AbgBuAGUAYwB0AGUAZAAnAH0ACgAJAAkACQAJAAkACQAJAB8ENQRABDUEPwQ%2BBDQEOgQ7BE4ERwQ4BEIETARBBE8ECgAJAAkACQAJAAkACQB7ADoAZQBsAHMAZQAgAGkAZgAgAHYAaQBlAHcAbQBvAGQAZQBsAC4AbABvAGEAZABpAG4AZwB9AAoACQAJAAkACQAJAAkACQAfBD4ENAQ6BDsETgRHBDUEPQQ4BDUELgAuAAoACQAJAAkACQAJAAkAewA6AGUAbABzAGUAfQAKAAkACQAJAAkACQAJAAkAHwQ%2BBDQEOgQ7BE4ERwQ4BEIETARBBE8ECgAJAAkACQAJAAkACQB7AC8AaQBmAH0ACgAJAAkACQAJAAkAPAAvAGIAdQB0AHQAbwBuAD4ACgAJAAkACQAJADwALwBmAG8AcgBtAD4ACgAJAAkACQB7AC8AaQBmAH0ACgAJAAkAPAAvAGQAaQB2AD4ACgAJADwALwBBAGMAYwBvAHIAZABpAG8AbgAuAEMAbwBuAHQAZQBuAHQAPgAKADwALwBBAGMAYwBvAHIAZABpAG8AbgAuAEkAdABlAG0APgAKAA%3D%3D&language=svelte)

### Expected result

shouldn't warn since it's svelte syntax

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
